packages/sui-domain/README.md

```diff
@@ -55,9 +55,9 @@ const EntryPoint = EntryPointFactory({useCases})
 const domain = new EntryPoint({config})
 ```
 
-### Injecting a logger to our domain
+### Injecting a logger or pde to our domain
 
-In order to increase the observability of your domain, you can pass a custom logger to your domain instance and it will be injected on each UseCase factory. Following the example of previous section:
+In order to increase the observability of your domain, you can pass a custom logger/pde to your domain instance and it will be injected on each UseCase factory. Following the example of previous section:
```

Comment:
I wouldn't say `pde` has anything to do with the _observability_ of your domain... perhaps having two different descriptions / entries on the readme would help.

Reply:
Nice, I will change it. Do you have some proposal for it?

Reply:
```suggestion
In order to increase the observability of your domain, you can pass a custom loggerto your domain instance and it will be injected on each UseCase factory. Following the example of previous section:
```

I'd leave the previous text as it was, and I would add the following:

```
In order to allow your domain to work with experiments and feature flags, you can pass a custom pde to your domain instance and it will be injected on each UseCase factory.
```